### ISSUE TITLE
teleporting into colshape may trigger enter/leave more often

### ISSUE THREAD

**Issue by eiksch** [altmp/altv-issues#672]
**Client/server version**

`client: alt:V build #2.1-dev4, branch dev`
`server: alt:V Server 2.1-dev2`

**Current behavior**

when teleporting into colshape, you have a chance to get multiple enter/leave events. walking in/out works fine

**Expected behavior**

only get one leave/enter event per teleport

**Steps to reproduce**

1. create colshape on your position
```
let col = new alt.ColshapeCircle(player.pos.x, player.pos.y, 10); //also happens with sphere
col.dimension = 0;
col.playersOnly = true;
```

2. setup logs
```
alt.on('entityEnterColshape', (cols, e)=>{console.log("entered Col", e.name, e.pos)});
alt.on('entityLeaveColshape', (cols, e)=>{console.log("leave Col", e.name, e.pos)});
```

3. teleport out `player.pos = /* somewhere else */`
4. teleport in `player.pos = /* colshape pos */`

repeat 3 and 4, until you get enter-leave-enter logs instead of a single enter log

**Context (environment)**

-

**Possible solution**

-

**Comment by tastydev** [comment]
can confirm

**Comment by C0kkie** [comment]
Will be fixed in dev/2.2-dev5
